helm/hpcc/values.schema.json

```diff
@@ -544,6 +544,9 @@
             "waitForMount": {
               "type": "boolean"
             },
+            "validatePlaneScript": {
+              "type": "array"
```

Comment:
Should include a description about how this is used.  Also is it worth restricting the element type:
` "items": { "type": "string" }`

Does it need an associated documentation jira. (And similar for exception hooks, are they documeted?)

Reply:
exceptionHandler are documented in helm/hpcc/docs/expert.md - probably enough?


validatePlaneScript isn't - and it feels like an expert setting too, perhaps it should go under a new expert: level under the plane to make it clear it is of that ilk - and then be documented in expert.md ..

Reply:
>Should include a description about how this is used. Also is it worth restricting the element type:
 "items": { "type": "string" }

will change and add a description